### PR TITLE
add history without definitions

### PR DESCRIPTION
**Pull request by ginnyfahs**

### CODE REVIEW

**Comment by ginnyfahs** [comment]
@TallOrderDev

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
 # phase-0-gps-1
 GPS 1.1
+git clone https://github.com/ginnyfahs/phase-0-gps-1.git
+   89  cd phase-0-gps-1
+   90  ls -a
+   91  subl .git
+   92  touch awesome_page.md
+   93  ls
+   94  git push origin master
```
Comment (TallOrderDev) [inline README.md]: This is awesome

File: README.md [hunk 2]
```diff
@@ -1,2 +1,21 @@
...
+   95  git add awesome_page.md
+   96  git status
+   97  git commit -m "add awesome file"
+   98  git push origin master
+   99  git checkout --help
+  100  git --help
+  101  git add --help
```
Comment (TallOrderDev) [inline README.md]: This is bad.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,21 @@
...
+   89  cd phase-0-gps-1
+   90  ls -a
+   91  subl .git
+   92  touch awesome_page.md
+   93  ls
+   94  git push origin master
+   95  git add awesome_page.md
+   96  git status
+   97  git commit -m "add awesome file"
+   98  git push origin master
```
Comment (TallOrderDev) [inline README.md]: This is another bad line

**Pull request merged**